src/swell/configuration/jedi/interfaces/geos_atmosphere/observations/mhs_metop-b.yaml

```
@@ -9,7 +9,7 @@ obs space:
       type: H5File
       obsfile: '{{cycle_dir}}/{{experiment_id}}.mhs_metop-b.{{window_begin}}.nc4'
   simulated variables: [brightnessTemperature]
-  channels: &mhs_metop-b_channels 1-5
+  channels: &mhs_metop-b_available_channels {{mhs_metop-b_avail_channels}}
```

The templating doesn't handle `-`, so all you need to do is remove it

```suggestion
  channels: &mhs_metop-b_available_channels {{mhs_metopb_avail_channels}}
```